### DISCUSSION TITLE
How do I run the streamlit version?

### DISCUSSION THREAD

**Discussion by JamesIV4** [hlky/stable-diffusion-webui#706] [Q&A] [answered]
Hey @ZeroCool940711, maybe you can help (or anyone else), I'm trying to run your streamlit version (which was merged into the dev branch I see), but I don't see any instructions on how to do so.

I cloned the dev repo into my folder that contains the main repo. Is that right? I'm unsure how to launch from there.

**Comment by ZeroCool940711** [comment, marked as answer]
@JamesIV4, you have to place the `webui_streamlit.py` file inside the script folder and the `webui_streamlit.yaml` inside the `config/webui` folder, once they are in those places just and assuming you have followed the instructions to run the main repo you just need to run `streamlit run scripts\webui_streamlit.py` from the root folder and it will launch the streamlit version of the UI. If you want more help you can go to the discord server for the repository where I have a channel for the streamlit webui and I can help you there better. 
[This is the discord link for the server.](https://discord.gg/gyXNe4NySY)

**Comment by dudumaroja** [comment]
assuming we need to place the prompts --precision full --no-half  to make images not... green. where we do put thoses for streamlit:

**Reply by JamesIV4** [reply]
Try editing the cfg file and see if you can put them in there. Another option might be to make a shortcut to the cfg and edit the properties and add those in the Target field at the end.

**Reply by dudumaroja** [reply]
seens like you can edit the file configs/webui/webui_streamlit.yaml o creat a file userconfig_streamlit.yaml in the same folder with theses settings
